packages/office-ui-fabric-react/src/components/Sticky/Sticky.tsx

```diff
@@ -206,7 +225,7 @@ export class Sticky extends BaseComponent<IStickyProps, IStickyState> {
 
       if (this.canStickyTop) {
         const distanceToStickTop = this.distanceFromTop - this._getStickyDistanceFromTop();
-        isStickyTop = distanceToStickTop <= container.scrollTop;
+        isStickyTop = distanceToStickTop < container.scrollTop;
```

Comment:
Should we have similar changes for Footer below?

Reply:
I don't think so.  I believe keeping it as `>=` for isStickyBottom is imperative to have sticky footer work for a details list scenario.